code/game/gamemodes/umbrage/umbrage_unsorted.dm

```diff
@@ -0,0 +1,66 @@
+//The umbrage mutant species. They have no blood, don't need to breathe, are immune to radiation and viruses, and, most importantly, cannot wield guns. They also have night vision.
+/datum/species/umbrage
+	name = "Umbrage"
+	id = "umbrage"
+	darksight = 8
+	invis_sight = SEE_INVISIBLE_MINIMUM
+	sexes = 0
+	blacklisted = 1
+	say_mod = "chirps"
+	meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/shadow
+	species_traits = list(NOBREATH, NOBLOOD, RADIMMUNE, VIRUSIMMUNE, NOGUNS)
+	dangerous_existence = 1
+	var/datum/action/innate/shadow/darkvision/vision_toggle
+
+/datum/species/umbrage/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	vision_toggle = new
+	vision_toggle.Grant(C)
+
+/datum/species/umbrage/on_species_loss(mob/living/carbon/C)
+	. = ..()
+	if(vision_toggle)
+		vision_toggle.Remove(C)
+
+/datum/species/umbrage/spec_life(mob/living/carbon/human/H)
```

Comment:
Like I said, I don't know if species subtypes like existing, but I'll push it with the next commit.